#Fix wayland-client and wayland-scanner pc files
#base on http://git.yoctoproject.org/cgit/cgit.cgi/meta-freescale/tree/recipes-graphics/wayland/wayland/fixpathinpcfiles.patch?h=sumo

diff --git a/src/wayland-client.pc.in b/src/wayland-client.pc.in
index eef61da..8221a87 100644
--- a/src/wayland-client.pc.in
+++ b/src/wayland-client.pc.in
@@ -1,7 +1,7 @@
 prefix=@prefix@
 exec_prefix=@exec_prefix@
 datarootdir=@datarootdir@
-pkgdatadir=@datadir@/@PACKAGE@
+pkgdatadir=${pc_sysrootdir}@datadir@/@PACKAGE@
 libdir=@libdir@
 includedir=@includedir@

diff --git a/src/wayland-scanner.pc.in b/src/wayland-scanner.pc.in
index 7b2a4c9..feb5b64 100644
--- a/src/wayland-scanner.pc.in
+++ b/src/wayland-scanner.pc.in
@@ -2,7 +2,7 @@ prefix=@prefix@
 exec_prefix=@exec_prefix@
 datarootdir=@datarootdir@
 pkgdatadir=@datadir@/@PACKAGE@
-wayland_scanner=@bindir@/wayland-scanner
+wayland_scanner=wayland-scanner
